qiskit_aer/primitives/sampler.py

```diff
@@ -99,7 +101,9 @@ def _call(
         if seed is not None:
             run_options.setdefault("seed_simulator", seed)
 
-        # Prepare circuits and parameter_binds
+        is_shots_none = "shots" in run_options and run_options["shots"] is None
```

Comment:
So if `shots` is not in the run options it is not considered to be `None`? 🤔

Reply:
In the current implementation, the default shots value of backend will be used. (1024)
The documentation mentioned the priority of seed, but I didn't write anything about the priority of shots.